conf/ports.py

```diff
@@ -139,6 +158,14 @@ def init_port(self, idx, conf_mode):
                     print('Failed to create AF_XDP socket for {}. Exiting...'.format(name))
                     sys.exit()
 
+        if conf_mode == 'cndp':
+            try:
+                # Initialize kernel fastpath.
+                self.init_datapath(cndp=True)
+            except:
+                print('Failed to create CNDP/AF_XDP socket for {}. Exiting...'.format(name))
+                sys.exit()
+
```

Comment:
What about having this rigth before or after the `if conf_mode == 'dpdk'` statement?

Reply:
Ok